lib/sparkql/function_resolver.rb

```diff
@@ -53,6 +53,16 @@ class Sparkql::FunctionResolver
     :now => { 
       :args => [],
       :return_type => :datetime
+    },
+    :date => { 
+      :args => [[:field,:datetime]],
```

Comment:
This format's a bit weird... basically I wanted functions to support either a field, or the literal type, and handle things accordingly. Fields get delayed resolution, literals are processed on the fly as per usual.